.github/workflows/release.yaml

```diff
@@ -11,20 +11,16 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-
-      - name: Unshallow
-        run: git fetch --prune --unshallow
+        with:
+          fetch-depth: 0
 
       - name: Set up Go
         uses: actions/setup-go@v1
         with:
           go-version: 1.15.x
 
-      - name: Login to Docker Hub Registry
-        run: echo "${{ secrets.DOCKERHUB_PASSWORD }}" | docker login -u "${{ secrets.DOCKERHUB_USERNAME }}" --password-stdin
-
-      - name: Login to GitHub Package Registry
-        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u "${{ github.actor }}" --password-stdin
+      - name: Login to GitHub Container Registry
+        run: echo "${{ secrets.CR_PAT }}" | docker login ghcr.io -u "${{ secrets.CR_USER }}" --password-stdin
```

Comment:
Never used the CR_USER on the other images.

Reply:
Did you hard-code a user or did `${{ github.actor }}` work?

Reply:
Using the PAT, it knows which user is trying to access the registry. I used the PAT of @adfinisbot

Reply:
Correction:

https://github.com/adfinis-sygroup/back8sup/blob/master/.github/workflows/release-container-image.yaml#L53

`repository_owner`

Reply:
```
      - name: Login to GHCR
        uses: docker/login-action@v1
        with:
          registry: ghcr.io
          username: ${{ github.repository_owner }}
          password: ${{ secrets.CR_PAT }}
          ```

Reply:
I reread [the docs](https://docs.github.com/en/packages/guides/migrating-to-github-container-registry-for-docker-images) and they now state that we can use the GITHUB_TOKEN:

> GitHub Container Registry now supports GITHUB_TOKEN for easy and secure authentication in your workflows. If your workflow is using a personal access token (PAT) to authenticate to ghcr.io, then we highly recommend you update your workflow to use GITHUB_TOKEN.

Reply:
Oh well, then I can change all my private repos and `back8sup` as well as `debug-container`